ruby/inc/leatherman/ruby/api.hpp

```diff
@@ -356,6 +356,10 @@ namespace leatherman {  namespace ruby {
          */
         VALUE (* const rb_hash_lookup2)(VALUE, VALUE, VALUE);
         /**
+		 * See MRI documentation.
+		 */
+		VALUE (* const rb_hash_lookup3)(VALUE, VALUE, VALUE, VALUE);
```

Comment:
compilation problem with rb_hash_lookup3 caused by its signature

Reply:
`rb_hash_lookup3` is not a function MRI implements.

`rb_hash_lookup2` is the same as `rb_hash_lookup` except it returns a custom default value if they key does not exist: https://github.com/ruby/spec/commit/bcb70e2eb309b7a8f17b42cd3f45c1d3c8f87e96